addon/components/listbox.hbs

```diff
@@ -0,0 +1,46 @@
+<div
+  {{did-insert this.registerDocumentClickListener}}
+  {{will-destroy this.unregisterDocumentClickListener}}>
+
+  {{yield
+    (hash
+      open=this.isOpen
+      disabled=(if @disabled true false)
```

Comment:
It seems like we define `disabled` here differently than we do when it's passed into the `Label` component on line 39. It would be good to be consistent about what `disabled` represents. Maybe we should have a property on the `Listbox` component instance that handles normalizing the argument into a Boolean, which we can reference with confidence everywhere we need access to that?

Reply:
https://github.com/GavinJoyce/ember-headlessui/pull/76/commits/4eca07ac5429cac6c5a7fb83c6e8de5b1bf5191e implements an internal `isDisabled` property which ensure disabled attribute consistency across the components.